internal/utils/main.go

```diff
@@ -337,16 +338,23 @@ func MustBucketFlags(flags *pflag.FlagSet, logger *EtlLogger) (path string) {
 	return
 }
 
-// MustGcsFlags gets the values of the bucket list specific flags: gcp-project and gcs-bucket
-func MustGcsFlags(flags *pflag.FlagSet, logger *EtlLogger) (bucket, credentials string) {
-	bucket, err := flags.GetString("gcs-bucket")
+// MustCloudStorageFlags gets the values of the bucket list specific flags: cloud-storage-bucket, cloud-credentials
+func MustCloudStorageFlags(flags *pflag.FlagSet, logger *EtlLogger) (bucket, credentials, provider string) {
```

Comment:
By making this method required, if a user only wants to read/write the files locally and deal with transmission in another process, are they going to get a warning every time they export without specifying a storage destination?

Reply:
I almost wonder if we make writing to a data store something you opt into, like the `--testnet` parameter and then pass accompanying cloud provider flags

Reply:
> Well the funny thing is the precedent in the code is to make it required but give it a default value 😆

yeah, yeah i know 😅 

i like that idea, i'm fine incrementally improving this interface so we can clean it up once we refactor stellar-etl to use a different ledgerbackend interface (ie, reading from GCS)